### ISSUE TITLE
recording accuracy from training and validation data

### ISSUE THREAD

**Issue by Fab1900** [jakeret/tf_unet#130]
Hi,
During training the U-net the loss and the accuracy from training images are calculated and recorded. Additionally I would like to calculate accuracy and loss from images for validation during the training which are not part of the training dataset.
So I have to split the data into training- and validation datak, that works fine. 
In the iteration loop I get the images for training batch_x, batch_y and the images for validation test_x and test_y. With the method "output_minibatch_stats" I want to calculate also accuracy and loss for the validation data. 
Now my question: How do I have to change the code, that the network computes the predictions for batch_x AND test_x when I start the session?

**Comment by fschi** [comment]
Did you solve this problem?

**Comment by Fab1900** [comment]
No I just decided to record the loss and the other values of the validation set because these are the important information to decide when the weights are trained enough.
